elements/nuxeo-document-actions/nuxeo-document-parent-inspector-button.js

```diff
@@ -0,0 +1,241 @@
+                <table class="table">
+                  <tr>
+                    <td>Tile:</td>
+                    <td>[[document.title]]</td>
+                  </tr>
+                  <tr>
+                    <td>Path:</td>
+                    <td>[[document.path]]</td>
+                  </tr>
+                  <template is="dom-if" if="[[currentUser.isAdministrator]]">
+                    <tr>
+                      <td>UID:</td>
+                      <td>[[document.uid]]</td>
+                    </tr>
+                  </template>
+                </table>
+              </div>
+            </div>
+            <div>
+              <h1>Facets:</h1>
+              <div class="facets">
+                <template is="dom-repeat" items="[[facets]]" as="facets">
+                  <div class="show-items">[[facets]]</div>
+                </template>
+              </div>
+            </div>
+
+            <template is="dom-if" if="[[currentUser.isAdministrator]]">
+              <div>
+                <h1>Schemas:</h1>
+                <div class="schemas">
+                  <template is="dom-repeat" items="[[schemas]]" as="schemas">
+                    <div class="show-items">[[schemas.prefix]]:[[schemas.name]]</div>
+                  </template>
+                </div>
+              </div>
+            </template>
+          </div>
+          <div class="actions">
+            <paper-button
+              noink
+              class="secondary"
+              dialog-dismiss
+              on-tap="_closeDialog"
+              role="button"
+              tabindex="0"
+              animated=""
+              elevation="0"
+              aria-disabled="false"
+              >[[i18n('command.close')]]</paper-button
+            >
+          </div>
+        </div>
+      </div>
+    </nuxeo-dialog>
+  `,
+
+  is: 'nuxeo-document-parent-inspector-button',
+  behaviors: [I18nBehavior, FiltersBehavior],
+
+  properties: {
+    document: {
+      type: Object,
+      notify: true,
+    },
+    icon: {
+      type: String,
+      value: 'icons:info',
+    },
+    layout: {
+      type: String,
+      value: 'parent-inspector',
+    },
+    facets: {
+      type: Array,
+      value: [],
+    },
+    schemas: {
+      type: Array,
+      value: [],
+    },
+    _label: {
+      type: String,
+      computed: '_computeLabel(i18n)',
+    },
+  },
+
+  ready() {
+    this.$.nxcon.connect().then((user) => {
+      this.currentUser = user;
+    });
+  },
+
+  _isAvailable() {
+    return true;
+  },
+
+  get dialog() {
+    this.facets = this.document.facets;
+    this.schemas = this.document.schemas;
+    return this.root.getElementById(`${this.layout}-dialog`);
+  },
+
+  _openDialog() {
+    this.dialog.open();
+  },
+
+  _closeDialog() {
+    this.dialog.close();
+  },
+
+  _computeLabel() {
+    const x = this.i18n('parentInspectorButton.tooltip');
+    return x;
```

Comment:
We could do this in a single line:
```suggestion
    return this.i18n('parentInspectorButton.tooltip');
```

Reply:
No need to store the translation in `x` if we don't use it. ⬆️